src/LintingViewExtension/LintingViewExtension.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using Dynamo.Extensions;
+using Dynamo.Graph;
+using Dynamo.Linting;
+using Dynamo.LintingViewExtension.Properties;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.Extensions;
+
+namespace Dynamo.LintingViewExtension
+{
+    public class LintingViewExtension : ViewExtensionBase, IExtensionStorageAccess
+    {
+        private const string EXTENSION_NAME = "Dynamo Linter";
```

Comment:
If they have not already, it would probably be good to consult with UX on naming for this extension.

Reply:
Yea mostly a placeholder name for now, was going to ask @nate-peters about this.

Reply:
Yeah, we need to make a final call on this... I still think calling it a "Linter" in a user-facing way feels wrong. Let me talk to a few of the designers and I'll get back to you by the end of the day.

Reply:
Let's call this "Graph Status", as opposed to "Dynamo Linter". Elizabeth and I looked at the design for a bit this morning and agreed that we'll probably end up adding more functionality to this extension over time, so we should keep the name broad for now. And we can leave "Graph Type" as-is.